src/page-ui.js

```diff
@@ -0,0 +1,224 @@
+import { retrieveProjects } from './localStorage';
```

Comment:
## FIX inability to create a project or todo
- [ ] Currently, the todos and projects are storing data sent from the UI to the `localStorage`, good job implementing this feature. However, after which, the page is expected to rerender and show/display the newly created project/todos, but it appears the user, get stuck from the initial stage of project/todo creation. Please fix this.

This is what we currently have 👇 
![todo-list](https://user-images.githubusercontent.com/22345961/101969141-e0f3ba00-3c33-11eb-8a66-23a898810d14.png)